src/k2/httpproxy/HTTPProxy.cpp

```diff
@@ -362,6 +362,125 @@ seastar::future<std::tuple<Status, CollectionCreateResponse>> HTTPProxy::_handle
     });
 }
 
+inline seastar::future<nlohmann::json> JsonResponse(Status&& status) {
```

Comment:
Not a big deal, but I don't think inline does anything useful here or in the following function

Reply:
Removed inline. Not sure if inline or not inline better.